java/ql/test/stubs/jshell/jdk/jshell/Snippet.java

```diff
@@ -0,0 +1,31 @@
+package jdk.jshell;
```

Comment:
Stubs solely used in experimental belong in `experimental/stubs`

(also update `options`)

Reply:
@smowton Please review again.